### PR TITLE
add metadata to gemspec

### PR DESCRIPTION
**Pull request by dwradcliffe**
Including new changelog link!

### CODE REVIEW

File: bundler.gemspec [hunk 1]
```diff
@@ -17,6 +17,13 @@ Gem::Specification.new do |s|
   s.homepage    = "http://bundler.io"
   s.summary     = "The best way to manage your application's dependencies"
   s.description = "Bundler manages an application's dependencies through its entire life, across many machines, systematically and repeatably"
+  
+  s.metadata    = {
+    "homepage_uri" => "https://bundler.io/",
+    "source_code_uri" => "http://github.com/bundler/bundler/",
+    "changelog_uri" => "https://github.com/bundler/bundler/blob/master/CHANGELOG.md",
```
Comment (segiddins) [inline bundler.gemspec]: Should this link to the current versions changelog instead of master?

Comment (dwradcliffe) [inline bundler.gemspec]: I think either option is okay but if you want that I'm happy to change it.

Comment (segiddins) [inline bundler.gemspec]: How about we link to master, but with the id of the version?

Comment (dwradcliffe) [inline bundler.gemspec]: Looks like the id has the date in it, I don't think we have that in the gemspec

**Comment by segiddins** [comment]
@indirect r?

**Comment by indirect** [comment]
looks good, thanks @dwradcliffe!

File: bundler.gemspec [hunk 2]
```diff
@@ -18,6 +18,13 @@ Gem::Specification.new do |s|
   s.summary     = "The best way to manage your application's dependencies"
   s.description = "Bundler manages an application's dependencies through its entire life, across many machines, systematically and repeatably"
 
+  s.metadata    = {
```
Comment (segiddins) [inline bundler.gemspec]: need to wrap in an `if s.respond_to?(:metadata=)`

Comment (segiddins) [inline bundler.gemspec]: ping

**Comment by segiddins** [comment]
@bundlerbot r+

**Comment by indirect** [comment]
@bundlerbot retry

**Pull request merged**